src/auto/font_metrics.rs

```diff
@@ -17,10 +17,6 @@ glib_wrapper! {
 }
 
 impl FontMetrics {
-    pub fn new() -> FontMetrics {
```

Comment:
Why did this disappear?

Reply:
`pango_font_metrics_new` removed from .gir
`Font` have `pango_font_get_metrics` so IMHO it was just not needed

Reply:
Ok